tools/asg-hevc/include/frame_processor.h

```diff
@@ -149,7 +149,14 @@ class FrameProcessor
     mfxU8 GetDefaultWeightedPredSample(mfxI32 predSampleLX);
     mfxU8 GetDefaultWeightedPredSample(mfxI32 predSampleL0, mfxI32 predSampleL1);
 
+#ifdef __clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-private-field"
+#endif
```

Comment:
Why we just can't remove m_FrameType?

Reply:
I found a variable that was used with the same name, but this variable is of a different class. So now I have removed `m_FrameType`

Reply:
build fails...